### ISSUE TITLE
Allow students to see their dashboard of modules and lessons

### ISSUE THREAD

**Issue by amellie** [Ali-Jahankah/bugsquashers-edu-app#25]
## Dependency

Depends on #24 

## Story

As a student
I want to see all of the lessons
So that I can see all the lessons available

## Description

When a student logs into the app, they should be able to see all the lessons grouped by modules

## Acceptance Criteria
The student dashboard should show all lesson names grouped by modules (show module name as heading), example:

Module 1: Caring for Ourselves
Lesson 1: Cleanliness and Hygine
Lesson 2: XXXX
Lesson 3: YYYY

Be creative, you can create a box for each lesson

Please note that the pages to each lesson will be done in a separate issue #26. If this issue has been done, please add a link for each lesson to the lesson page

**Comment by amellie** [comment]
@ShadiSafaee I am not sure if we are able to add a custom sorting mechanism since we only have 1 more week. Therefore, would you be able to show the modules and lessons, sorted by their names please? At least, this temporarily solves the problem of custom sorting
